src/Sylius/Bundle/ShopBundle/EventListener/UserRegistrationListener.php

```diff
@@ -74,6 +74,6 @@ private function enableAndLogin(ShopUserInterface $user): void
         $this->userManager->persist($user);
         $this->userManager->flush();
 
-        $this->userLogin->login($user, $this->firewallContextName);
+        $this->security->login($user, $this->firewallContextName);
```

Comment:
```suggestion
        $this->security->login($user, null, $this->firewallContextName);
```
Firewall name is the 3rd argument in this method